readme.md

```diff
@@ -4,6 +4,8 @@
 Join us on [Bozeman Laravel Slack]
 
 ## Getting Started
+Note: If you don't already, please follow the instructions from the Laravel Documentation to install [Homestead.](https://laravel.com/docs/6.x/homestead) 
```

Comment:
Should this say "If you don't already have Homestead (or valet), please follow..."

Reply:
Maybe we should consider adding Laravel Valet as an option here. In my opinion it's a better option than Homestead for most people.

Reply:
Yeah, we could include more options for getting set up. Valet doesn't work on Windows or Linux, for us non-Mac weirdos 😛